### PR TITLE
EPMRPP-49154: [AGENT] Jasmine. Report launches/ suites/ tests/ steps with attachments

### PR DESCRIPTION
**Pull request by KatsiarynaTatarynovich**
works with EPMRPP-49154--logs-with-suite-params

### CODE REVIEW

File: lib/jasmine-reportportal-reporter.js [hunk 1]
```diff
@@ -67,9 +75,40 @@ class ReportportalReporter {
...
     }
 
+    addTestItemLog(testItemLog) {
+        if (testItemLog && testItemLog.suite) {
+            const logs = (this.suiteLogs.get(testItemLog.suite) || []).concat([Object.assign(testItemLog.log, { time: this.getTime() })]);
+
+            this.suiteLogs.set(testItemLog.suite, logs);
+        } else {
+            const logs = this.additionalCustomParams.logs
+                ? { logs: this.additionalCustomParams.logs.concat([testItemLog.log]) }
```
Comment (ElenaRomanchuk) [inline lib/jasmine-reportportal-reporter.js:85]: What's about fixation time of particular log?

File: lib/constants/testStatuses.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  PASSED: 'passed',
+  FAILED: 'failed',
+  SKIPPED: 'skipped',
+  PENDING: 'pending',
```
Comment (ElenaRomanchuk) [inline lib/constants/testStatuses.js:5]: I suggest we should divide RP statuses and Jasmine statuses.

File: lib/jasmine-reportportal-reporter.js [hunk 3]
```diff
@@ -67,9 +75,40 @@ class ReportportalReporter {
...
+    sendLaunchLog(log) {
+        this.sendLog(this.tempLaunchId, log);
+    }
+
+    sendLog(tempId, { level, message = '', file, time }) {
+        this.client.sendLog(tempId,
+            {
+                message,
+                level,
+                time: time || this.getTime()
```
Comment (AmsterGet) [inline lib/jasmine-reportportal-reporter.js:101]: This field will be added automaticaly by client. Isn't it?

File: lib/jasmine-reportportal-reporter.js [hunk 4]
```diff
@@ -76,14 +76,16 @@ class ReportportalReporter {
     }
 
     addTestItemLog(testItemLog) {
+        const logsWithTime = Object.assign(testItemLog.log, { time: this.getTime() });
```
Comment (AmsterGet) [inline lib/jasmine-reportportal-reporter.js:79]: Is it logs or one log?

Comment (KatsiarynaTatarynovich) [reply lib/jasmine-reportportal-reporter.js]: log, fixed it

**Pull request merged**